README.md

```diff
@@ -249,7 +249,7 @@ Refer  below for next steps when any indicator reports a `false`.
 | S0008        | Determines if there is at least 20% disk free on the codedir data partition.        | See [documentation](https://portal.perforce.com/s/article/Check-for-space-code-deployment-fails?name=000007241)                                                                                    |                                                                                                                                                                                                                                       |
-| S0009        | Determines if pe-puppetserver service is running and enabled on relevant components. | Checks that the service can be started and enabled by running `puppet resource service pe-puppetserver ensure=running`, examines `/var/log/puppetlabs/puppetserver/puppetserver.log` for failures.                                                                                                                                                                                                                                                                                               | If you are unable to explain the service outage from the logging, or are unable to start the service, open a Support ticket referencing S0009 and provide the `/var/log/puppetlabs/puppetserver/puppetserver.log`, showing an unsuccessful startup.                                                                              |
+| S0009        | Determines if pe-puppetserver service is running and enabled on relevant components. | See [documentation](https://portal.perforce.com/s/article/Check-if-pe-puppetserver-service-is-running-enabled)                                                                                                                                                                                                                                                                                               | If you are unable to explain the service outage from the logging, or are unable to start the service, open a Support ticket referencing S0009 and provide the `/var/log/puppetlabs/puppetserver/puppetserver.log`, showing an unsuccessful startup.                                                                              |
```

Comment:
I think it would be better to ask for a support script instead of an individual log file.  Please change the language asking for the logfile to something like the other indicators that ask for a sup script, e.g. 

> provide the output of the [support script.](https://puppet.com/docs/pe/latest/getting_support_for_pe.html#pe_support_script)